dynamodb-example/main.go

```diff
@@ -0,0 +1,112 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"math/rand"
+	"strconv"
+	"time"
+
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-lambda-go/lambda"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
+)
+
+// Response is of type APIGatewayProxyResponse since we're leveraging the
+// AWS Lambda Proxy Request functionality (default behavior)
+//
+// https://serverless.com/framework/docs/providers/aws/events/apigateway/#lambda-proxy-integration
+type Response events.APIGatewayProxyResponse
+
+// Handler is our lambda handler invoked by the `lambda.Start` function call
+func Handler(request events.APIGatewayProxyRequest) (events.APIGatewayProxyResponse, error) {
+
+	// Create DynamoDB client
+	sess := session.Must(session.NewSessionWithOptions(session.Options{
+		SharedConfigState: session.SharedConfigEnable,
+	}))
+	svc := dynamodb.New(sess)
+
+	// Create DynamoDB table if it doesn't exist
+	tableName := os.Getenv("DYNAMODB_TABLE")
```

Comment:
@gjgd je te propose de créer un fichier .env.example pou documenter tt les variables d'environnement. SI tu as eu aussi besoin de mettre tes AWS credentials en variables d'env faudrait l'indiquer aussi

Chez nous on utilisais une sorte de syntaxe Yardoc pour documenter, ça donne un truc du genre

```
##
# Website configuration
#
# HOST your server is running under
#   So an absolute URL can be generated:
#   - For emails sent by Action mailer.
#   - For links sent to slack/intercom
#   - Do not forget HTTPS if you need it
# @example "localhost"
# @example "https://www.myjobglasses.com"
HOST=null
#
# @example "localhost"
# @example "https://aphrodite-staging.myjobglasses.com"
FRONTEND_HOST=null
```

Reply:
environment variable are defined in `serverless.yml`. 

For example, DYNAMO_DB is defined here: https://github.com/gjgd/dont-slack-evil/blob/89153054de9b327bcbd9d95c406c2ee957735173/serverless.yml#L8

Reply:
OK for documenting the variables in a `.env.example` file. (and of course, write in the README that they are documented in this file :) )